platform/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ target_compile_definitions(${LIB_NAME} PUBLIC
     CPU_MIMXRT1051DVL6B_cm7
     CPU_MIMXRT1051DVL6B
     __FPU_PRESENT
+    __MCUXPRESSO
```

Comment:
do you know why this is needed?

Reply:
The `system_MIMXRT1051.c` uses this define for the init vector table in `SystemInit()` function.
```
#if defined(__MCUXPRESSO)
    extern uint32_t g_pfnVectors[];  // Vector table defined in startup code
    SCB->VTOR = (uint32_t)g_pfnVectors;
#endif
```